inst/@sym/isempty.m

```diff
@@ -60,7 +60,6 @@
 %! a = sym ([1 2]);
 %!assert (~isempty (sym (1)))
 %!assert (isempty (sym (se)))
-%!assert (isempty (se == []))
```

Comment:
Oops, I missed that this was still here before I merged :(

I will continue this discussion at #650